src/Tests/dotnet-workload-install.Tests/GivenDotnetWorkloadInstall.cs

```diff
@@ -126,16 +126,17 @@ public void GivenWorkloadInstallItCanUpdateInstalledManifests()
         [Fact]
         public void GivenWorkloadInstallItCanDownloadToOfflineCache()
```

Comment:
Could you add a test to make sure it is the manifest in cache get used(to find the packs)? and later installed (placed on disk)